packages/vm/tests/util.ts

```diff
@@ -30,10 +30,10 @@ export function dumpState(state: any, cb: Function) {
 
   function readStorage(state: any, account: Account) {
```

Comment:
Isn't it possible to just make this function `async`?

Reply:
I just realized that if we do this it doesn't change the function signature so that would make a lot of sense. I do think that currently this function mimics what an async function does except it is not marked as `async` function.